editor/src/tool/tools/select.rs

```diff
@@ -173,20 +173,31 @@ impl Fsm for SelectToolFsmState {
 						DrawingBox
 					};
 					buffer.into_iter().rev().for_each(|message| responses.push_front(message));
+
+					// ToDo: Clean up cloning
+					let ignore_layers = match data.bounding_box_path.as_ref() {
+						Some(bounding_box) => Vec::from([bounding_box.to_vec()]),
+						None => Vec::new(),
```

Comment:
Can these be simplified with the `vec![]` macro? I'm actually not at all sure what the `Vec::from([bounding_box.to_vec()])` even does.